seed.py

```diff
@@ -0,0 +1,98 @@
+####
+# Seed Databases
+
+from faker import Faker
```

Comment:
I think we might need to add this to requirements.txt if this was something that got added with pip3

Reply:
Yes it needs to be added to the requirements.txt  I did a compare of the other lines I had in my requirements and I believe this was the only thing different.